[2024-11-18T12:36:22.993+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T12:36:23.022+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_dag.check_environment scheduled__2024-11-17T00:00:00+00:00 [queued]>
[2024-11-18T12:36:23.039+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_dag.check_environment scheduled__2024-11-17T00:00:00+00:00 [queued]>
[2024-11-18T12:36:23.039+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-18T12:36:23.106+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): check_environment> on 2024-11-17 00:00:00+00:00
[2024-11-18T12:36:23.117+0000] {standard_task_runner.py:72} INFO - Started process 145 to run task
[2024-11-18T12:36:23.122+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline_dag', 'check_environment', 'scheduled__2024-11-17T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp8415lack']
[2024-11-18T12:36:23.126+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask check_environment
[2024-11-18T12:36:23.212+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline_dag.check_environment scheduled__2024-11-17T00:00:00+00:00 [running]> on host 3145f9671393
[2024-11-18T12:36:23.370+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline_dag' AIRFLOW_CTX_TASK_ID='check_environment' AIRFLOW_CTX_EXECUTION_DATE='2024-11-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-17T00:00:00+00:00'
[2024-11-18T12:36:23.371+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T12:36:23.417+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-18T12:36:23.418+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n            echo "Current working directory: $PWD" &&\n            echo "DBT_PROFILES_DIR: $DBT_PROFILES_DIR" &&\n            echo "Listing dbt project directory:" &&\n            ls -la /opt/airflow/etl_pipeline &&\n            echo "DBT version:" &&\n            dbt --version\n        ']
[2024-11-18T12:36:23.435+0000] {subprocess.py:99} INFO - Output:
[2024-11-18T12:36:23.439+0000] {subprocess.py:106} INFO - Current working directory: /tmp/airflowtmpc9z4rr79
[2024-11-18T12:36:23.440+0000] {subprocess.py:106} INFO - DBT_PROFILES_DIR:
[2024-11-18T12:36:23.440+0000] {subprocess.py:106} INFO - Listing dbt project directory:
[2024-11-18T12:36:23.449+0000] {subprocess.py:106} INFO - total 72
[2024-11-18T12:36:23.450+0000] {subprocess.py:106} INFO - drwxrwxrwx 11 1000 1000 4096 Nov 18 10:18 .
[2024-11-18T12:36:23.450+0000] {subprocess.py:106} INFO - drwxrwxrwx  8 root root 4096 Nov 18 12:36 ..
[2024-11-18T12:36:23.451+0000] {subprocess.py:106} INFO - -rwxrwxrwx  1 1000 1000   29 Nov  8 18:16 .gitignore
[2024-11-18T12:36:23.452+0000] {subprocess.py:106} INFO - -rwxrwxrwx  1 root root   41 Nov 18 08:21 .user.yml
[2024-11-18T12:36:23.452+0000] {subprocess.py:106} INFO - -rwxrwxrwx  1 1000 1000  571 Nov  8 18:16 README.md
[2024-11-18T12:36:23.453+0000] {subprocess.py:106} INFO - drwxrwxrwx  2 1000 1000 4096 Nov  8 18:16 analyses
[2024-11-18T12:36:23.453+0000] {subprocess.py:106} INFO - drwxrwxrwx  3 1000 1000 4096 Nov 16 07:19 dbt_packages
[2024-11-18T12:36:23.454+0000] {subprocess.py:106} INFO - -rwxrwxrwx  1 1000 1000 1380 Nov 13 12:54 dbt_project.yml
[2024-11-18T12:36:23.454+0000] {subprocess.py:106} INFO - drwxrwxrwx  2 1000 1000 4096 Nov 10 06:26 logs
[2024-11-18T12:36:23.455+0000] {subprocess.py:106} INFO - drwxrwxrwx  2 1000 1000 4096 Nov  8 18:16 macros
[2024-11-18T12:36:23.456+0000] {subprocess.py:106} INFO - drwxrwxrwx  5 1000 1000 4096 Nov 13 11:29 models
[2024-11-18T12:36:23.456+0000] {subprocess.py:106} INFO - -rwxrwxrwx  1 1000 1000  113 Nov 16 07:19 package-lock.yml
[2024-11-18T12:36:23.457+0000] {subprocess.py:106} INFO - -rwxrwxrwx  1 1000 1000   60 Nov 16 07:18 packages.yml
[2024-11-18T12:36:23.457+0000] {subprocess.py:106} INFO - -rwxrwxrwx  1 1000 1000  196 Nov 18 10:18 profiles.yml
[2024-11-18T12:36:23.458+0000] {subprocess.py:106} INFO - drwxrwxrwx  2 1000 1000 4096 Nov  8 18:16 seeds
[2024-11-18T12:36:23.458+0000] {subprocess.py:106} INFO - drwxrwxrwx  2 1000 1000 4096 Nov  8 18:16 snapshots
[2024-11-18T12:36:23.459+0000] {subprocess.py:106} INFO - drwxrwxrwx  4 1000 1000 4096 Nov 11 11:39 target
[2024-11-18T12:36:23.459+0000] {subprocess.py:106} INFO - drwxrwxrwx  2 1000 1000 4096 Nov  8 18:16 tests
[2024-11-18T12:36:23.460+0000] {subprocess.py:106} INFO - DBT version:
[2024-11-18T12:36:24.407+0000] {local_task_job_runner.py:127} ERROR - Received SIGTERM. Terminating subprocesses
[2024-11-18T12:36:24.417+0000] {process_utils.py:132} INFO - Sending 15 to group 145. PIDs of all processes in the group: [147, 149, 145]
[2024-11-18T12:36:24.418+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 145
[2024-11-18T12:36:24.419+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-11-18T12:36:24.425+0000] {subprocess.py:117} INFO - Sending SIGTERM signal to process group
[2024-11-18T12:36:24.444+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=149, status='terminated', started='12:36:23') (149) terminated with exit code None
[2024-11-18T12:36:24.449+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 257, in execute
    result = self._run_inline_command(bash_path=bash_path, env=env)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 284, in _run_inline_command
    return self.subprocess_hook.run_command(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/subprocess.py", line 104, in run_command
    for raw_line in iter(self.sub_process.stdout.readline, b""):
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3095, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-11-18T12:36:24.465+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=etl_pipeline_dag, task_id=check_environment, run_id=scheduled__2024-11-17T00:00:00+00:00, execution_date=20241117T000000, start_date=20241118T123623, end_date=20241118T123624
[2024-11-18T12:36:24.466+0000] {local_task_job_runner.py:127} ERROR - Received SIGTERM. Terminating subprocesses
